README

```diff
@@ -134,23 +134,14 @@ CD-ROM: My CD-ROM doesn't work.
 
```

Comment:
(minor fix at  line 49): `If you have your game on a cdrom you may try ...`
Rename `cdrom` to `CD-ROM` to match with consistent naming in the documentation.